src/demo/helpers.js

```diff
@@ -212,3 +212,13 @@ export const finishLoading = (startTime, onComplete) => {
 
   setTimeout(onComplete, delayTime);
 };
+
+export const setPageAndSendPageview = page => {
```

Comment:
I think it would be fine to leave "setPage" as an internal detail and just call this something like "reportPageView".